src/components/listing-create.js

```diff
@@ -135,12 +139,15 @@ class ListingCreate extends Component {
 
   async onSubmitListing(formListing, selectedSchemaType) {
     try {
-      console.log(formListing)
       this.setState({ step: this.STEP.METAMASK })
-      const transactionReceipt = await origin.listings.create(formListing.formData, selectedSchemaType)
+      console.log(formListing)
```

Comment:
Is this supposed to be here?

Reply:
I just moved it, from above the state change, I guess. 🤷‍♂️

Reply:
ok cool! I don't like console.log statements to be in source code but that's a battle I'll fight another day haha.

Reply:
Uh, right, so don't look at the messaging-orbit branch 🙈

Reply:
uh oh